Kai documentation michal revisions

jerab111 opened the pull request:
I believe:
- Users don't read sequentially - they scan for what they need NOW. That's why it's better to split content in more pages  (getting started; use cases; best practises; troubleshooting) and edit the structure
- Every extra section = 20% higher abandonment


- index.md - aimed for more user friendly approach + initial visual.
- use-cases - aimed for different approach - content might need work
- best-practises - really well done by Jordan!
-added troubleshooting - we need it, but the content is pure claude's work
- added security and privacy.md - some content there might be junk, but we 100% need it
- deleted technical.md - i don't think we need it

jordanrburger committed:
Add Kai - AI Assistant section to navigation and main AI page

jerab111 committed:
Refactor KAI assistant documentation for clarity

Updated references from 'Kai' to 'KAI' and improved descriptions for clarity and context awareness. Added sections on usage and support.

jerab111 committed:
Update getting-started.md

jerab111 committed:
Revise 'Getting Started' guide to include use cases

Updated the 'Getting Started' guide for Kai, changing the title and permalink, and enhancing the content to include detailed use cases and examples for data engineering tasks.

jerab111 committed:
Update getting-started.md

jerab111 committed:
Enhance Kai use cases with tips and examples

Updated use cases for Kai assistant with additional tips and examples for data engineering tasks.

jerab111 committed:
Update best-practices.md

jerab111 committed:
Create troubleshooting.md

jerab111 committed:
Create security-and-privacy.md

jerab111 committed:
Delete ai/kai-assistant/technical.md

davidesner commented on ai/kai-assistant/index.md:
```diff
@@ -1,77 +1,84 @@
+|---------|------------------|-----|
+| **Context** | No access to your project | Full access to configs, schemas, job logs |
+| **Actions** | Can only suggest code | Creates, modifies, and runs components |
+| **Authentication** | Manual setup required | Already authenticated in platform |
```
If we are comparing generic AI Assistant with no MCP than authentication is not a valid difference.

jerab111 committed:
Revise KAI assistant documentation for branding updates

Updated the KAI assistant documentation to reflect branding changes and improve clarity.
